plugins/package-managers/gradle-plugin/src/main/kotlin/OrtModelBuilder.kt

```diff
@@ -186,14 +182,21 @@ internal class OrtModelBuilder : ToolingModelBuilder {
                             }
 
                             val modelBuildingResult = poms.getValue(id.toString())
+                            val dependencies = if (id in visitedDependencies) {
```

Comment:
Can you provide more details on the issue, I'm not sure which of the two options below apply for the example of a "a -> b -> a" circle:

Before: "a"
After: "a -> b"

or

Before: "a -> b"
After: "a -> b -> a"

Reply:
With this change it's like in your second example now.

Reply:
I'm not sure if other package managers cut the circle like in the first example, but we discuss this separately.